.github/workflows/llm_performance_tests.yml

```diff
@@ -13,11 +13,9 @@ on:
     branches: [main]
     paths:
       - ".github/workflows/llm_performance_tests.yml"
-      - ".github/workflows/llm-binary-build.yml"
-      - ".github/actions/llm/setup-llm-env/action.yml"
-      - ".github/actions/llm/remove-llm-env/action.yml"
-      - ".github/actions/llm/download-llm-binary/action.yml"
-      - "python/llm/test/benchmark/**"
+      - "python/llm/test/benchmark/arc-perf-test.yaml"
```

Comment:
should include other yamls as well since this workflow also contains spr perf?

Reply:
ok